### DISCUSSION TITLE
Can't import spaCy on Raspberry Pi 4b

### DISCUSSION THREAD

**Discussion by alfuwu** [explosion/spaCy#13131] [Help: Installation]
When I try to use spaCy on my Raspberry Pi, it throws the following error:
```
ImportError: /home/pi/.local/lib/python3.9/site-packages/blis/cy.cpython-39-arm-linux-gnueabihf.so: undefined symbol: __atomic_load_8
```

**Comment by adrianeboyd** [comment]
Hi, I see that there's an older related issue here, where it looks like using a 64-bit OS with 64-bit python solved the problem: https://github.com/explosion/spaCy/issues/6638

That issue is almost 3 years old, and since then we've added binary wheels for 64-bit `aarch64` for all our packages, so you also typically wouldn't need to compile our packages from source anymore.

(As far as I'm aware spacy v3 (not spacy v2) should be compatible with 32-bit python, but I'm afraid we aren't currently able to provide any further support for use with 32-bit python.)

**Reply by louisabraham** [reply]
Hello! I cannot find those binary wheels for `aarch64` on https://pypi.org/project/spacy/

**Reply by honnibal** [reply]
For the latest versions of spaCy we've simplified our build system around cibuildwheel, so we don't have to host our own Github Actions runners. However, Github doesn't yet support Linux ARM workers for public repos. This should be coming soon, but in the meantime we're missing Linux ARM wheels for the latest spaCy versions. If you do `pip install spacy --only-binary` you should get the latest version that has wheels for your platform.

**Reply by louisabraham** [reply]
Oh great! thanks!
